Sasview 1079

rprospero opened the pull request:
I've fixed the Corfunc perspective so that both the Q-Space extrapolation and the real space transform are both plotted.  Additionally, the plots are now included alongside the dialog on the right, instead of being squeezed in at the top, which make the plots easier to read.  Finally, I've added axes labels to the plots for additional clarity.

Ignore to odd colour scheme in the imags below.  That's a known issue with my computer alone.

Before

![before](https://user-images.githubusercontent.com/1728853/54922091-54551580-4eff-11e9-8056-875c73471d55.png)

After

![after](https://user-images.githubusercontent.com/1728853/54922102-5cad5080-4eff-11e9-8365-6c462d17219a.png)

rprospero commented:
I've gone ahead and removed all of the limits from the corfunc transform, so that we don't need to worry about extending it further down the road.  Instead, there's a reasonable default plotting range and, more importantly, I've added the navigation toolbars so that the user can explore more of the plot range if they desire.

![2019-03-26-094953_1918x1031_scrot](https://user-images.githubusercontent.com/1728853/54987468-8f615280-4fac-11e9-8b4a-c3ed90bef2f9.png)

rozyczko commented on src/sas/qtgui/Perspectives/Corfunc/CorfuncPerspective.py:
```diff
@@ -6,6 +6,7 @@
 # global
 from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg \
     as FigureCanvas
+from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT
```
We don't use the Qt toolbar elsewhere, instead relying on our own Plotter class with a number of context menu items.

rozyczko replied:
This might not be a serious issue, considering the relevant charts are added as indices to the dataset in the Data Explorer and can be viewed with "Create New" or "Quick Plot"

rozyczko commented on src/sas/qtgui/Perspectives/Corfunc/CorfuncPerspective.py:
```diff
@@ -77,14 +84,17 @@ def draw_real_space(self):
         self.axes.set_yscale("linear")
+        self.axes.set_xlabel("Z [$\AA$]")
+        self.axes.set_ylabel("Correlation")
+        self.fig.tight_layout()
```
Would also be great if the plots had titles as it makes it difficult to tell which graph is showing what, especially when no data is loaded.

rprospero commented:
I've fixed the vertical spacing and added tiles to the plots, as well as a legend on the data plot.  I did also remove the matplotlib toolbars.  I took a quick look at replacing the two plots with a proper plotting widget and saw that it would be a significant rewrite, but, as you pointed out, the results are still in the data explorer.

![2019-03-26-151006_1918x1031_scrot](https://user-images.githubusercontent.com/1728853/55008766-64d9be80-4fd9-11e9-85e9-57a459db30a2.png)
